ifrs/core/models.py

```diff
@@ -50,27 +50,27 @@ class FileMetadata(BaseModel):
             + " crypt4GH envelope)."
         ),
     )
-    encrypted_part_size: int = Field(
+    encrypted_part_sizes: list[int] = Field(
         ...,
         description=(
-            "The size of the file parts of the encrypted content (excluding the"
-            + " crypt4GH) as used for the encryption_parts_md5 and the"
-            + " encryption_parts_sha256 in bytes. The same part size is recommended for"
+            "The sizes of the file parts of the encrypted content (excluding the"
+            + " crypt4GH envelope) as used for the encrypted_parts_md5 and the"
```

Comment:
```suggestion
            + " crypt4gh envelope) as used for the encrypted_parts_md5 and the"
```

I don't care if we use `crypt4gh` or `crypt4GH`, be would should probably keep it consistent - at least throughout the document.